helm.el

```diff
@@ -3114,12 +3119,35 @@ and `helm-pattern'."
              (if (string-match "[[:upper:]]" pattern) nil t)))
     (t helm-case-fold-search)))
 
-(defun helm-match-from-candidates (cands matchfns match-part-fn limit source)
-  (let (matches)
+
+(defun helm--mapconcat-initials-pattern (pattern seperators)
```

Comment:
So this is designed to not need flx?  That's really interesting.  I will follow along and see how it goes.